### PR TITLE
Test for loading cache after dirty shutdown

### PR DESCRIPTION
**Pull request by katlapinka**
Needs:
https://github.com/Open-CAS/open-cas-linux/pull/658

Signed-off-by: Katarzyna Lapinska <katarzyna.lapinska@intel.com>

### CODE REVIEW

**Review by Ostrokrzew — DISMISSED** [review]

File: test/functional/tests/lazy_writes/test_dirty_load.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    with TestRun.step("Load cache."):
+        cache = casadm.load_cache(cache_dev)
+
+        caches_num = len(casadm_parser.get_caches())
+        if caches_num != 1:
+            TestRun.LOGGER.error(f"Wrong number of caches. Expected 1, actual {caches_num}.")
+
+        cores_num = len(casadm_parser.get_cores(cache.cache_id))
+        if cores_num != 1:
```
Comment (rafalste) [inline test/functional/tests/lazy_writes/test_dirty_load.py:73]: Didn't you add two cores in lines 39-40?

Comment (rafalste) [inline test/functional/tests/lazy_writes/test_dirty_load.py:73]: Didn't you add two cores in lines 39-40?

File: test/functional/tests/lazy_writes/test_dirty_load.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            TestRun.LOGGER.error(f"Wrong number of cores. Expected 1, actual {caches_num}.")
```
Comment (rafalste) [inline test/functional/tests/lazy_writes/test_dirty_load.py:74]: caches_num -> cores_num

File: test/functional/tests/lazy_writes/test_dirty_load.py (hunk 1 above)
Comment (Ostrokrzew) [reply test/functional/tests/lazy_writes/test_dirty_load.py:73]: One is removed in lines 58-59.

File: test/functional/tests/lazy_writes/test_dirty_load.py (hunk 2 above)
Comment (katlapinka) [reply test/functional/tests/lazy_writes/test_dirty_load.py:74]: done

**Review by Ostrokrzew — APPROVED** [review]

**Review by rafalste — APPROVED** [review]

**Pull request merged**